### PR TITLE
Added the location update as a step

### PR DESCRIPTION
**Pull request by Rohit25negi**
Ticket: https://dimagi-dev.atlassian.net/browse/IIO-612
Run update location in parallel to other tasks.

### CODE REVIEW

**Comment by Rohit25negi** [comment]
this must not be merged without https://github.com/dimagi/commcare-hq/pull/26752

File: dags/dashboard_subdags.py [hunk 1]
```diff
@@ -55,6 +55,13 @@ def generate_uuid():
         dag=monthly_dag
     )
 
+    _update_aggregate_locations_tables = BashOperator(
```
Comment (calellowitz) [inline dags/dashboard_subdags.py]: no reason to prefix this with an underscore

File: dags/dashboard_subdags.py [hunk 2]
```diff
@@ -171,13 +178,22 @@ def generate_uuid():
     )
 
     get_agg_id >> create_aggregation_record >> daily_attendance
+    # Running awc_location build step in parallel to other steps
```
Comment (calellowitz) [inline dags/dashboard_subdags.py]: nit: i don't think these comments (or the ones below) add much, but they aren't necessarily a bad thing

**Comment by Rohit25negi** [comment]
> This is definitely an improvement, but I think this can actually be better. Not blocking since it wasn't in the ticket, but I think this only needs to be run once per aggregation (not once per month)

yay!!! Same thing I had mentioned in the ticket as comment. I think we can take that as a next step to quantitatively measure the gain we get with each of them.

**Pull request merged**